perses/tests/test_samplers.py

```diff
@@ -154,10 +154,187 @@ def run_samplers(testsystem_names, niterations=5):
+                custom_force = hybrid_system.getForce(force_dict['CustomNonbondedForce_sterics'])
+                nonbonded_force = hybrid_system.getForce(force_dict['NonbondedForce_sterics'])
+                custom_force.setUseLongRangeCorrection(False)
+                nonbonded_force.setUseDispersionCorrection(True)
+
+                # Set up repex simulation
+                reporter_file = os.path.join(temp_dir, f"{wt_name}-{mutant_name}.nc")
+                reporter = MultiStateReporter(reporter_file, checkpoint_interval=10)
+                hss = HybridRepexSampler(mcmc_moves=mcmc.LangevinSplittingDynamicsMove(timestep=4.0 * unit.femtoseconds,
+                                                                                       collision_rate=1.0 / unit.picosecond,
+                                                                                       n_steps=250,
+                                                                                       reassign_velocities=True,
+                                                                                       n_restart_attempts=20,
+                                                                                       splitting="V R R R O R R R V",
+                                                                                       constraint_tolerance=1e-06),
+                                                                                       replica_mixing_scheme='swap-all',
+                                        hybrid_factory=htf,
+                                        online_analysis_interval=None)
+                hss.setup(n_states=12, temperature=300 * unit.kelvin, t_max=300 * unit.kelvin,
+                          storage_file=reporter, endstates=False)
+                hss.energy_context_cache = cache.ContextCache(capacity=None, time_to_live=None, platform=platform)
+                hss.sampler_context_cache = cache.ContextCache(capacity=None, time_to_live=None, platform=platform)
+
+                # Run simulation
+                hss.extend(n_iterations)
+
+                # Retrieve free energy
+                reporter.close()
+                reporter = MultiStateReporter(reporter_file)
+                analyzer = MultiStateSamplerAnalyzer(reporter, max_n_iterations=n_iterations)
+                f_ij, df_ij = analyzer.get_free_energy()
+                data[f"{wt_name}-{mutant_name}"] = {'free_energy': f_ij[0, -1], 'error': df_ij[0, -1]}
+
+        assert np.isclose(np.array([data['arg-ala']['free_energy'] + data['ala-arg']['free_energy']]), np.array([data['lys-ala']['free_energy'] + data['ala-lys']['free_energy']]))
+
 
-if __name__=="__main__":
-    for t in test_hybrid_scheme():
-        t()
+# if __name__=="__main__":
+    # for t in test_hybrid_scheme():
+    #     t()
 #    for t in test_samplers():
 #        print(t.description)
 #        t()
+#     run_RESTCapableHybridTopologyFactory_repex()
```

Comment:
We should probably remove these lines and also the `test_hybrid_scheme()` commented function, if we are not using them. In any case the comments are already in the history in case we ever need them again, I guess.
```suggestion
```